### PR TITLE
Make datafeeder header configurable

### PR DESCRIPTION
**Pull request by tkohr**
PR moves the header iframe from the `index.html` to the `app.component` and allows defining its `headerHeight` and `headerSrc` from the settings (which are fetched from the datadir if present, cf `frontend-config.json`)

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Thanks Tobias !
Calling @fgravin or @jahow for a quick review.

**Comment by tkohr** [comment]
There seems to be an issue with prettier somewhere (on this branch?). When I fix files detected with `npm run format:check` others are detected next. `npm run format` touches on more than 50 files, which is strange, but may be necessary. Any idea why?

**Comment by tkohr** [comment]
Hum, I also tried with prettier version `2.5.1` we use upstream. `npm run prettier` still wants to touch on a lot of files, which is not the case on `upstream/master`. Wondering if I'm missing a difference in the config somewhere?

**Comment by fgravin** [comment]
> Hum, I also tried with prettier version `2.5.1` we use upstream. `npm run prettier` still wants to touch on a lot of files, which is not the case on `upstream/master`. Wondering if I'm missing a difference in the config somewhere?

There have been an Angular upgrade since this branch.
If you work locally with the same `node_modules`, then you'll have inconsistency.

I would say that format must be ok upstream, before merging upstream in this repository.

File: apps/datafeeder/src/settings.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { environment } from './environments/environment'
 const SETTING_API = `${environment.apiUrl}/config/frontend`
 
 class Settings {
+  headerHeight = '90px'
```
Comment (fgravin) [inline apps/datafeeder/src/settings.ts:7]: How those settings are substituted from georchestra data_dir ?

Comment (tkohr) [reply apps/datafeeder/src/settings.ts:7]: If I saw right, the [settings](https://github.com/georchestra/datadir/blob/master/datafeeder/frontend-config.json) are fetched from the datadir [here](https://github.com/georchestra/geonetwork-ui/blob/bd6bf17df4e762d69b1e9bd73fd9ded7f2db27ab/apps/datafeeder/src/settings.ts#L43).

**Comment by tkohr** [comment]
> There have been an Angular upgrade since this branch. If you work locally with the same `node_modules`, then you'll have inconsistency.
> 
> I would say that format must be ok upstream, before merging upstream in this repository.

What is strange, is that the issue also appears in the CI. Locally, I've also tried with a repo that only tracks this one. Do you think we should merge upstream into this repo?

**Comment by tkohr** [comment]
I've opened #5 regarding the breaking build.

**Comment by tkohr** [comment]
CI has been fixed by #5

**Comment by fvanderbiest** [comment]
Up ? Thanks !

**Review by jahow — APPROVED** [review]
Looking good, thanks!

**Pull request merged**